amd_openvx_extensions/CMakeLists.txt

```diff
@@ -27,12 +27,12 @@ find_package(FFmpeg QUIET)
 find_package(AMDRPP QUIET)
 
 if(GPU_SUPPORT)
+    find_package(miopengemm PATHS ${ROCM_PATH} QUIET)
```

Comment:
make sure to modify the setup to install the MIOpen for the right backend

Reply:
Yes, issue #572 is for adding the support for different backend into the setup script. The setup script changes will be added in a separate PR.